.circleci/config.yml

```diff
@@ -236,6 +236,60 @@ jobs:
             RELEASE_CHANNEL: experimental
           command: ./scripts/circleci/run_devtools_e2e_tests.js
 
+  run_fixtures_flight_tests:
+    docker:
+      - image: cimg/openjdk:20.0-node
+    environment: *environment
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      # Fixture copies some built packages from the workroot after install.
+      # That means dependencies of the built packages are not installed.
+      # We need to install dependencies of the workroot to fulfill all dependency constraints
+      - setup_node_modules
+      - restore_cache:
+          name: Restore yarn cache of fixture
+          keys:
+            - v2-yarn_cache_fixtures_flight-{{ arch }}-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install fixture dependencies
+          working_directory: fixtures/flight
+          command: |
+            yarn install --frozen-lockfile --cache-folder ~/.cache/yarn
+            if [ $? -ne 0 ]; then
+              yarn install --frozen-lockfile --cache-folder ~/.cache/yarn
+            fi
+      - save_cache:
+          name: Save yarn cache of fixture
+          key: v2-yarn_cache_fixtures_flight-{{ arch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
+      - run:
+          working_directory: fixtures/flight
+          name: Playwright install deps
+          command: |
+            npx playwright install
+            sudo npx playwright install-deps
+      - run:
+          working_directory: fixtures/flight
+          command: |
+            yarn build
+            # Need to build twice for unknown reasons
```

Comment:
Otherwise we get https://app.circleci.com/pipelines/github/facebook/react/50479/workflows/7727b75f-e654-4558-9294-a2a710b8ebbd/jobs/783757/steps

Which the fixture also catches

Reply:
I believe if you set `FAST_REFRESH=false`, you can skip the build step completely. This is how I verified this locally: `rm -rf build ./node_modules/.cache && FAST_REFRESH=false yarn test`. My understanding is, that this avoids needing to transform the `node_modules/react-refresh` ES modules.

Reply:
On the other hand, this would just mask the issues that users might run into when trying to run the flight fixture locally. Is fast refresh that useful for the fixture? If not, we could also disable it in `fixtures/flight/config/env.js`, and not only in the CI.